ras_backstage/controllers/collection_instrument_controller.py

```diff
@@ -0,0 +1,27 @@
+import json
+import logging
+
+from structlog import wrap_logger
+
+from ras_backstage import app
+from ras_backstage.common.requests_handler import request_handler
+
+logger = wrap_logger(logging.getLogger(__name__))
+
+
+def upload_collection_instrument(survey_id, collection_exercise_id, file):
+    logger.debug('Uploading collection instrument', collection_exercise_id=collection_exercise_id, survey_id=survey_id)
+    url = f'{app.config["RAS_COLLECTION_INSTRUMENT_SERVICE"]}' \
+          f'collection-instrument-api/1.0.2/upload/{collection_exercise_id}'
```

Comment:
Is the CE API version likely to change? If so, it might be worth adding it as a config / env variable.

Reply:
Unsure but I'd guess if it was to change it would be because of a breaking change so might need code changes anyway and at that point is it worth being an environmental variable?